full-stack-asset-transfer-guide/applications/trader-typescript/src/contract.ts

```diff
@@ -76,8 +76,10 @@ export class AssetTransfer {
     }
 
     async transferAsset(id: string, newOwner: string, newOwnerOrg: string): Promise<void> {
-        // TODO: Implement me!
-        // Submit a 'TransferAsset' transaction, which requires [id, newOwner, newOwnerOrg] arguments.
+        console.log(`transferring asset '${id}' to ${newOwner}, ${newOwnerOrg}`);
+        // TODO: implement the transferAsset() function.
+        // TODO: submit a 'TransferAsset' transaction, requiring [id, newOwner, newOwnerOrg] arguments.
+        return Promise.reject(new Error('TODO: implement the contract.ts transferAsset() function.'));
```

Comment:
Should just be able to throw the error and Node.js will work out the rest - it's a function that is already marked as async